recipe/meta.yaml

```diff
@@ -125,6 +128,20 @@ outputs:
       commands:
         - python -c "import ray; ray.init()"
 
+  - name: ray-default
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('ray-core', exact=True) }}
+        - colorful
+
+    test:
+      imports:
+        # there doesn't appear to be a k8s module, not sure how to test this
+        - ray
```

Comment:
Please improve this comment.

Reply:
Removed (wrongly copied when creating `ray-default`)